DNN Platform/Library/UI/Skins/SkinPackageInfo.cs

```diff
@@ -79,9 +87,10 @@ public string SkinName
             }
         }
 
-        [XmlIgnore]
+        /// <inheritdoc cref="ISkinPackageInfo.Skins"/>
+        [XmlIgnore]
         [JsonIgnore]
-        public Dictionary<int, string> Skins
+        public List<SkinInfo> Skins
```

Comment:
Is this intended to be a breaking change?

Reply:
You were too fast; I was still typing my comment 😉

But yes, it's a breaking change. I didn't see any usage of Skins in DNN; but for extensions this is a breaking change.
`SkinInfo` exists in the DNN library but is never used. With this change we can future proof the Skins table.

I can make my own Dictionary and update the skins objects based on what is changed, and renamed this property to `SkinObjects`.

Reply:
Note, if these changes can't be approved because we rather have backward combability than future proofing, I can revert this change without a problem.

Reply:
Mayby we can have SkinList and deprecate Skin